src/BoundingBox3D.cs

```diff
@@ -94,5 +91,27 @@ public Vector3 GetCenter()
             var z = (ZMax + ZMin) / 2;
             return new Vector3((float)x, (float)y, (float)z);
         }
+
+        public double[] GetBoundingvolumeBox() {
```

Comment:
box we don't use anymore? only region?

Reply:
box is still used for mapbox export, mapbox3dtiles currently does not support region